src/main/java/io/github/cottonmc/cotton/config/ConfigManager.java

```diff
@@ -65,21 +66,21 @@
                 return result;
             }
```

Comment:
don't know why braces style is bad here, let's keep `} catch (IOException e) {` on one line